docs/register/parameters.rst

```diff
@@ -69,7 +69,14 @@ Parameters with Lambda Expression Components
 
 With lambda expression component registrations, rather than passing the parameter value *at registration time* you enable the ability to pass the value *at service resolution time*. (:doc:`Read more about resolving with parameters. <../resolve/parameters>`)
 
-In the component registration expression, you can make use of the incoming parameters by changing the delegate signature you use for registration. Instead of just taking in an ``IComponentContext`` parameter, take in an ``IComponentContext`` and an ``IEnumerable<Parameter>``:
+In the component registration expression, you can make use of the incoming parameters by using the generic lambda ``Register`` methods:
```

Comment:
Should we change the parameter to be something that isn't a `string`? I ask because usually folks don't register a string as a dependency.

```c#
// Generally not a thing
builder.RegisterInstance("configValue");
```

I see that it's a string below in the parameter section, but it may be worth updating both examples to be some sort of configuration object that might be more commonly registered.

Reply:
I've given a more "config object" style example.